oracle/config/transfer-watcher.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
   },
   event: 'Transfer',
   eventFilter: { to: process.env.COMMON_FOREIGN_BRIDGE_ADDRESS },
+  sender: 'home',
```

Comment:
should it be the foreign side here? The oracle listens `Transfer` events of an erc20 token to send the confirmations to the home side

Reply:
As you said, confirmation is send on the home side, that's why the sender should be configured to `home`. The only place where it is set to `foreign`, is for a collected-signatures watcher.

Reply:
Right. Missed that it was for the sender.